src/components/Table/GridTable.tsx

```diff
@@ -477,6 +494,7 @@ function renderTable<R extends Kinded>(
   filteredRows: RowTuple<R>[],
   firstRowMessage: string | undefined,
   stickyHeader: boolean,
+  maxCardPadding: number | undefined,
```

Comment:
Wondering if we can even support nested cards via the <table> approach anymore? Are we using maxCardPadding inside this anywhere?

Reply:
Via what approach? I think Github lost part of it, oh like the `table` approach?

No, nested cards aren't really supported here, but the param is necessary, even if its unused, b/c right now all of the rendertable / rendercss / rendervirtual methods must have the same method signatures.